### PR TITLE
Update man pages for 2.11

### PR DESCRIPTION
**Pull request by eepp**

### CODE REVIEW

File: doc/man/lttng-enable-rotation.1.txt [hunk 1]
```diff
@@ -30,13 +30,14 @@ so that an automatic rotation occurs every time the total size of the
...
-With both the option:--timer and option:--size options, LTTng checks the
+For both option:--timer and option:--size options, LTTng checks the
 schedule condition periodically using the monitor timers of the tracing
 session's channels. This means that, with the option:--timer option, the
 automatic rotation can occur when the elapsed time since the last
 automatic rotation is greater than 'PERIOD', and with the option:--size
 option, the automatic rotation can occur when the size of the flushed
-part of the current trace chunk is greater than 'SIZE'. See the
+part of the current trace chunk is greater than 'SIZE'. Therefore the
+'PERIOD' and 'SIZE' arguments are approximations. See the
```
Comment (jgalar) [inline doc/man/lttng-enable-rotation.1.txt]: While `SIZE` is an approximation, the precision of which is governed by the `monitor-timer`, the `PERIOD` is fairly precise as it basically relies on the precision of the platform's implementation of POSIX timers.

File: doc/man/lttng-enable-event.1.txt [hunk 2]
```diff
@@ -127,11 +137,10 @@ application's execution, only one event is emitted.
...
 
-* Tracepoint/system call name ('EVENT' argument with
-  option:--tracepoint or option:--syscall options) or
-  dynamic probe/function name/address
-  (option:--probe or option:--function option's argument) which must
-  match event source's equivalent.
+* Tracepoint/system call name ('EVENT' argument with option:--tracepoint
+  or option:--syscall options) or dynamic probe/function name/address
+  (option:--probe, option:--userspace-probe, and option:--function
+  option's argument) which must match event source's equivalent.
```
Comment (jgalar) [inline doc/man/lttng-enable-event.1.txt]: Is there a word missing on the last line?

Comment (eepp) [reply doc/man/lttng-enable-event.1.txt]: event source's equivalent [tracepoint/system call name] (implicit).

File: doc/man/lttng-enable-event.1.txt [hunk 3]
```diff
@@ -410,8 +410,10 @@ Dynamic user space probes
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 With the option:--userspace-probe option, you can instrument function
 entries of any user space binary (application or library) using either
-an available symbol name or a SystemTap SDT probe's provider and probe
-names.
+an available symbol name or a SystemTap Statically Defined Tracepoints
+(SDT, a DTrace-style marker) probe's provider and probe names. As of
+this version, only SDT probes that are :not: surrounded by a reference
+counter (semaphore).
```
Comment (jgalar) [inline doc/man/lttng-enable-event.1.txt]: Is there a "are supported" missing here?

Comment (eepp) [reply doc/man/lttng-enable-event.1.txt]: Fixed in 9a49aca396e28c579e2fd143693edb917f39631b.

Comment (eepp) [reply doc/man/lttng-enable-event.1.txt]: Fixed in 22337deae7794dcd623888e9beee89d886272165.

**Comment by jgalar** [comment]
Merged in master and stable-2.11, thanks!

**Pull request closed without merging**